### PR TITLE
[IMP] orm/changelog: store translations in jsonb

### PR DESCRIPTION
**Pull request by HydrionBurst**

### CODE REVIEW

File: content/developer/reference/backend/orm/changelog.rst [hunk 1]
```diff
@@ -4,6 +4,15 @@
 Changelog
 =========
 
+Odoo Online version 16.0
```
Comment (rco-odoo) [inline content/developer/reference/backend/orm/changelog.rst:7]: The "online" versions are the SaaS versions.
```suggestion
Odoo version 16.0
```

**Review by rco-odoo — APPROVED** [review]
@robodoo r+

**Comment by rco-odoo** [comment]
@AntoineVDV can you merge, please?

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**